docs/website/docs/ml-frameworks/tensorflow-lite.md

```diff
@@ -1,14 +1,61 @@
 # TensorFlow Lite Integration
 
-!!! todo
-    [Issue#5455](https://github.com/google/iree/issues/5455): write this documentation
+IREE supports compiling and running pre-trained TFLite models.  It converts a model to [TOSA MLIR](https://mlir.llvm.org/docs/Dialects/TOSA/), then compiles it into a VM module.
+
+## Prerequisites
+
+Download a pre-trained TFLite model from the list of [hosted models](https://www.tensorflow.org/lite/guide/hosted_models).
+
+
+Install IREE pip packages, either from pip or by
+[building from source](../building-from-source/optional-features.md#building-python-bindings):
+
+```shell
+python -m pip install \
+  iree-compiler-snapshot \
+  iree-runtime-snapshot \
+  iree-tools-tflite-snapshot \
+  -f https://github.com/google/iree/releases
+```
+
+## Importing models
+
+Fist, import the TFLite model to TOSA MLIR:
 
-<!-- TODO(??): overview (mention TOSA?) -->
+```shell
+iree-import-tflite \
+  sample.tflite \
+  -o sample.mlir
+```
 
-<!-- TODO(??): pip install vs build from source -->
+Next, compile the TOSA MLIR to a VM flatbuffer, using either the command line tools or the [Python API](https://google.github.io/iree/bindings/python/):
 
-<!-- TODO(??): python model code vs tf lite flatbuffer -->
+#### Using the command-line tool
 
-<!-- TODO(??): Colab notebooks, code samples -->
+``` shell
+iree-translate \
+  --iree-mlir-to-vm-bytecode-module \
+  --iree-hal-target-backends=vmvx \
+  sample.mlir \
+  -o sample.vmfb
+```
+
+#### Using the python API
+
+``` python
+from iree.compiler import compile_str
+with open('sample.mlir') as sample_tosa_mlir:
+  compiled_flatbuffer = compile_str(sample_tosa_mlir.read(), target_backends=["vmvx"])
+```
+
+The flatbuffer can then be loaded to a VM module and run through IREE's runtime.
+
+## Samples
+
+| Colab notebooks |  |
+| -- | -- |
+Text classification with TFLite and IREE | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/iree/blob/main/colab/tflite_text_classification.ipynb)
+
+!!! todo
 
-<!-- TODO(??): using the tflite bindings -->
+    [Issue#3954](https://github.com/google/iree/issues/3954): Add documentation for an Android demo using the [TFLite bindings](https://google.github.io/iree/bindings/tensorflow-lite/), once it is complete at [not-jenni/iree-android-tflite-demo](https://github.com/not-jenni/iree-android-tflite-demo).
```

Comment:
We should introduce both the TensorFlow Lite C API and Java bindings.

Reply:
I updated the comment to mention the Java bindings.  Is there a good sample of the TensorFlow Lite C API that I could link to?

Reply:
https://github.com/google/iree/blob/main/bindings/tflite/smoke_test.cc is all we have so far